central/complianceoperator/v2/report/manager/manager_test.go

```diff
@@ -83,10 +89,13 @@ func (m *ManagerTestSuite) TestHandleReportRequest() {
 	identity.EXPECT().UID().AnyTimes().Return("user-id")
 	identity.EXPECT().FullName().AnyTimes().Return("user-name")
 	identity.EXPECT().FriendlyName().AnyTimes().Return("user-friendly-name")
+	m.bindingsDataStore.EXPECT().GetScanSettingBindings(gomock.Any(), gomock.Any()).AnyTimes()
```

Comment:
nit:  You could probably use some of the values for more precision instead of all the `gomock.Any()` calls.

Reply:
I added a few matchers to the mocks: See [54930e1](https://github.com/stackrox/stackrox/pull/13804/commits/54930e1ac8c478909cf872a6e920b9ad2e933365)